pkg/reconciler/taskrun/validate_resources_test.go

```diff
@@ -174,9 +181,7 @@ func TestValidateResolvedTaskResources_ValidParams(t *testing.T) {
 	}, {
 		Name: "myobj",
 		Value: *v1beta1.NewObject(map[string]string{
-			"key1":      "val1",
-			"key2":      "val2",
-			"extra_key": "val3",
+			"key2": "my value",
```

Comment:
could you add a test case where the default defines a value for one of the keys and that value is overridden by the taskRun?

Reply:
[this](https://github.com/tektoncd/pipeline/pull/5427/files#diff-e5039733a366256d36ca2680d4f7ea1fab71fd2fc78d0526ba9a13ba511af4a2R158-R202) might cover this case. Please let me know otherwise.